bin/cron/create_rollup_tables

```diff
@@ -46,6 +46,8 @@ def get_contained_question(contained_level)
 end
 
 def main
+  level_types = ['Multi', 'Match', 'EvaluationMulti']
```

Comment:
If you are going to extract this, please call it `answer_level_types` or something else a bit more descriptive